lib/croutons/breadcrumb_trail.rb

```diff
@@ -15,16 +15,16 @@ def initialize(template_identifer, objects = {})
       build_breadcrumbs
     end
 
-    def method_missing(name, *args)
+    def method_missing(name, *args, &block)
```

Comment:
The only reason I did not pass the block along in line 20 is that I am not sure that rails url helpers do take a block as argument. But thinking out loud, not passing the block might be a violation of "Tell Don't Ask". Any thoughts on that?